dandi/cli/cmd_shell_completion.py

```diff
@@ -36,6 +36,8 @@ def shell_completion(shell):
             raise click.UsageError(
                 "Could not determine running shell: SHELL environment variable not set"
             )
+        if shell.lower().endswith(".exe"):
```

Comment:
I wouldn't rely on it be just .exe -- couldn't it be `.com` (IIRC some "small executable fiting into RAM, e.g. a "driver" ;)) or some `.bat` shim? may be do some `splitext` and check if any of those?

Reply:
Done.